test/langtools/jdk/javadoc/doclet/testTagMisuse/TestTagMisuse.java

```diff
@@ -23,6 +23,7 @@
 
 /*
  * @test
+ * @bug 8035473 8288692
```

Comment:
I know that it's an urgent fix. However, consider either checking the output for the newly created "no reference" warning or delete 8288692 from @bug.  At the very least, update the copyright year.

Reply:
I think the bug number belongs in the bug because it is implicitly a test that javadoc does not crash.

Reply:
Updated copyright year